<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="skins.qrc">
    <normaloff>:/icons/skins/icons/MusicPlayer.ico</normaloff>:/icons/skins/icons/MusicPlayer.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#MainWidget{
    background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
}

/*顶层界面皮肤配置*/
QWidget#wgt_topLayer{
    min-height: 50px;
    max-height: 50px;
}

QPushButton#pb_close, #pb_min, #pb_max, #pb_setting{
    min-width: 17px;
    max-width: 17px;
    min-height: 16px;
    max-height: 16px;
}

QPushButton#pb_close{
    border-image: url(:/images/skins/images/closeNormal.png);
}

QPushButton#pb_close:pressed{
    border-image: url(:/images/skins/images/closePressed.png);
}

QPushButton#pb_min{
    border-image: url(:/images/skins/images/minimizeNormal.png);
}

QPushButton#pb_min:pressed{
    border-image: url(:/images/skins/images/minimizePressed.png);
}

QPushButton#pb_max{
    border-image: url(:/images/skins/images/maximizeNormal.png);
}

QPushButton#pb_max:pressed{
    border-image: url(:/images/skins/images/maximizePressed.png);
}

QPushButton#pb_setting{
    border-image: url(:/images/skins/images/setNormal.png);
}

QPushButton#pb_setting:pressed{
    border-image: url(:/images/skins/images/setPressed.png);
}

/*中间层界面皮肤配置*/
QWidget#wgt_middleLayer{
    border-image: url(:/icons/skins/icons/back6.png);
}

QListWidget#lw_songNameList{
    min-width: 250px;
    max-width: 250px;
    color: white;
    font: 14px &quot;微软雅黑&quot;;
    border: none;
    background-color: transparent;
}

QListWidget#lw_songNameList::item{
    min-height: 35px;
    max-height: 35px;
    background-color: transparent;
}

QListWidget#lw_songNameList::item:hover{
    color: rgb(248, 217, 129);
    background-color: transparent;
}

QListWidget#lw_songNameList::item:selected{
    color: red;
    text-decoration: underline;
    background-color: transparent;
}

QListWidget#lw_lyricShow{
    color: rgb(248, 217, 129);
    font: 75 14pt &quot;微软雅黑&quot;;
    border: none;
    background-color: transparent;
}

QListWidget#lw_lyricShow::item{
    min-height: 40px;
    max-height: 40px;
    background-color: transparent;
}

QListWidget#lw_lyricShow::item:selected{
    color: red;
    text-decoration: underline;
    background-color: transparent;
}

/*底层界面皮肤配置*/
QWidget#wgt_bottomLayer{
    min-height: 100px;
    max-height: 100px;
}
QLabel#lb_songName{
    min-width: 200px;
    max-width: 200px;
    min-height: 30px;
    max-height: 30px;
    font: 75 14pt &quot;微软雅黑&quot;;
    color: rgb(255, 255, 255);
}
QLabel#lb_position{
    font: 75 14pt &quot;微软雅黑&quot;;
    color: rgb(255, 255, 255);
}

QPushButton#pb_player{
    min-width: 40px;
    max-width: 40px;
    min-height: 40px;
    max-height: 40px;
    border-image: url(:/images/skins/images/playStartPressed.png)
}

QPushButton#pb_prev, #pb_next{
    min-width: 13px;
    max-width: 13px;
    min-height: 21px;
    max-height: 21px;
}

QPushButton#pb_prev{
    border-image: url(:/images/skins/images/previous.png);
}

QPushButton#pb_prev:pressed{
    border-image: url(:/images/skins/images/previousPressed.png);
}

QPushButton#pb_prev:hover{
    border-image: url(:/images/skins/images/previousHover.png);
}

QPushButton#pb_next{
    border-image: url(:/images/skins/images/next.png);
}

QPushButton#pb_next:pressed{
    border-image: url(:/images/skins/images/nextPressed.png);
}

QPushButton#pb_next:hover{
    border-image: url(:/images/skins/images/nextHover.png);
}

QLabel#lb_volume{
    min-width: 18px;
    max-width: 18px;
    min-height: 24px;
    max-height: 24px;
    border-image: url(:/icons/volume.png);
}

QLabel#lb_volume::menu-indicator{
    image:none;
}

QSlider::sub-page:horizontal{
    margin: 5px 0px 5px 0px;
    background-color: #3fd680;
}

QSlider#hs_volume{
    min-width: 80px;
    max-width: 80px;
    min-height: 13px;
    max-height: 13px;
}

QSlider#hs_volume:handle{
    border-image: url(:/images/skins/images/volumeHandle.png);
}

QSlider#hs_position{
    min-height: 13px;
    max-height: 13px;
}

QSlider#hs_position:handle{
    border-image: url(:/images/skins/images/volumeHandle.png);
}

QPushButton#pb_playMode{
    min-width: 30px;
    max-width: 30px;
    min-height: 25px;
    max-height: 25px;
    border-image: url(:/images/skins/images/loopNormal.png);
}


QPushButton#pb_desktopLyric{
    min-width: 25px;
    max-width: 25px;
    min-height: 25px;
    max-height: 25px;
    border-image: url(:/icons/skins/icons/lrc_normal.png)
}

QPushButton#pb_desktopLyric:hover{
    border-image: url(:/icons/skins/icons/lrc_hover.png)
}

QPushButton#pb_desktopLyric:pressed{
    border-image: url(:/icons/skins/icons/lrc_pressed.png)
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="wgt_topLayer" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lb_headline">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="skins.qrc">:/icons/skins/icons/widgetIcon.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_setting">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_min">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_max">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_close">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="wgt_middleLayer" native="true">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListWidget" name="lw_songNameList">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_11">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QListWidget" name="lw_lyricShow">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="wgt_bottomLayer" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="5">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="17">
         <widget class="QPushButton" name="pb_playMode">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="12">
         <widget class="QSlider" name="hs_volume">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="11">
         <widget class="QLabel" name="lb_volume">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="skins.qrc">:/icons/skins/icons/volume.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="pb_prev">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="9">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="8">
         <widget class="QPushButton" name="pb_next">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="lb_songName">
          <property name="text">
           <string>歌名</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="3">
         <widget class="QLabel" name="lb_headIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="skins.qrc">:/icons/skins/icons/huxia.png</pixmap>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QPushButton" name="pb_player">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="2" colspan="11">
         <widget class="QSlider" name="hs_position">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="14" rowspan="2" colspan="5">
         <widget class="QLabel" name="lb_position">
          <property name="text">
           <string>播放进度</string>
          </property>
         </widget>
        </item>
        <item row="0" column="13">
         <spacer name="horizontalSpacer_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="15">
         <widget class="QPushButton" name="pb_desktopLyric">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="16">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="skins.qrc"/>
 </resources>
 <connections/>
</ui>
